zebra-state/Cargo.toml

```diff
@@ -26,7 +26,7 @@ multiset = { git = "https://github.com/jmitchell/multiset", rev = "91ef8550b518f
 proptest = { version = "0.10.1", optional = true }
 proptest-derive = { version = "0.3", optional = true }
 regex = "1"
-rlimit = "0.5.4"
+rlimit = "0.7.0"
```

Comment:
The deny.toml CI failed due to temporary alpine linux Docker network errors.
We should restart those checks after a few hours.

This PR changes dependencies, so we need to make sure these checks pass.